internal/pkg/print/markdown/content/testdata/content-WithAggregateTypeDefaults.golden

```diff
@@ -0,0 +1,116 @@
+Usage:
+
+module "foo" {
+  source = "github.com/foo/bar"
+
+  id   = "1234567890"
+  name = "baz"
+
+  zones = ["us-east-1", "us-west-1"]
+
+  tags = {
+    Name         = "baz"
+    Created-By   = "first.last@email.com"
+    Date-Created = "20180101"
+  }
+}
+
+## Inputs
+
+These variables are defined:
+
+### unquoted
+
+Description: -
+
+Type: `string`
+
+Default: -
+
+### string-2
+
+Description: It's string number two.
+
+Type: `string`
+
+Default: -
+
+### string-1
+
+Description: It's string number one.
+
+Type: `string`
+
+Default: `bar`
+
+### map-3
+
+Description: -
+
+Type: `map`
+
+Default: `{}`
+
+### map-2
+
+Description: It's map number two.
+
+Type: `map`
+
+Default: -
+
+### map-1
+
+Description: It's map number one.
+
+Type: `map`
+
+Default: `{ "a": 1, "b": 2, "c": 3 }`
```

Comment:
I had considered this but the only reason I didn't do this was that it could have looked weird (the fact that the fence is _under_ "Default" not _in front of_ it:

---

### variable-1

Description: value-1 description

Type: `string`

Default: `foo`

### variable-2

Description: value-2 description

Type: `map`

Default: 
```
{
  "a": 1,
  "b": 2,
  "c": 3
}
```

### variable-3

Description: value-3 description

Type: `string`

Default: `bar`

---

Or may be I'm reading too much into this! :wink: It doesn't actually look very bad!

Reply:
I like it pretty much, actually :)

Reply:
Sure, pretty-printed code fence it is.

Reply:
This should be prettified now.